pages/apis/rest_api/analytics/flaky_tests.md

```diff
@@ -0,0 +1,46 @@
+# Flaky tests API
+
+The Flaky test API endpoint provides information about tests detected as flaky in a test suite.
+
+{:toc}
+
+## List all flaky tests
+
+Returns a [paginated list](<%= paginated_resource_docs_url %>) of the flaky tests detected in a test suite.
+
+```bash
+curl "https://api.buildkite.com/v2/analytics/organizations/{org.slug}/suites/{suite.slug}/flaky-tests"
+```
+Optional [query string parameters](/docs/api#query-string-parameters):
+<table class="fixed-width">
+  <tbody>
+    <tr>
+      <th>
+        <code>test_id</code>
+      </th>
+      <td>
+        <span>Filters the results for a specific test.</span>
+        <p class="Docs__api-param-eg"><em>Example:</em> <code>?test_id=my_test_id</code></p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+```json
+[
+  {
+    "id": ":test_id",
+    "web_url": "/organizations/my_great_org/analytics/suites/my_suite_name/tests/:test_id",
+    "scope": "My test scope",
+    "name": "My test name",
+    "location": null,
+    "file_name": null,
```

Comment:
Maybe we could provide a real-ish example?

```suggestion
    "scope": "User#email",
    "name": "is correctly formatted",
    "location": "./spec/models/user_spec.rb:42",
    "file_name": "./spec/models/user_spec.rb",
```

Or have fun with a humorous or pop-culture reference